cms/src/collections/collections.ts

```diff
@@ -0,0 +1,55 @@
+import {CmsCollection} from 'netlify-cms-core';
+import EXERCISE_FIELDS from '../fields/exercise';
+import CONTRIBUTORS_FIELDS from '../fields/contributors';
+import * as content from '../../../content/content.json';
+import {generateFilesCollectionFromi18nFiles} from '../lib/i18n';
+
+export const exercises: CmsCollection = {
+  name: 'exercises',
+  label: 'Exercises',
+  label_singular: 'exercise',
+  folder: '/content/src/exercises',
+  identifier_field: 'id',
+  extension: 'json',
+  format: 'json',
+  create: true,
+  delete: true,
+  publish: true,
+  summary: '{{fields.name}}',
+  slug: '{{id}}',
+  editor: {
+    preview: false,
+  },
+  fields: EXERCISE_FIELDS,
+  i18n: true,
+};
+
+export const contributors: CmsCollection = {
+  name: 'other',
+  label: 'Other',
+  files: [
+    {
+      label: 'All Contributors',
+      name: '.all-contributorsrc',
```

Comment:
This is a merge missmatch, this should be named `.all-contributorsrc`

Reply:
Are you not saying it should be named what it is named? 😂

Reply:
Hehe sorry. Remove the dot

Reply:
Will fix